LDKNodeMonday/View/Home/NetworkSettingsView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  NetworkSettingsView.swift
+//  LDKNodeMonday
+//
+//  Created by Daniel Nordh on 03/12/2024.
+//
+
+import LDKNode
+import SwiftUI
+
+struct NetworkSettingsView: View {
+    @Environment(\.dismiss) private var dismiss
+
+    @Bindable var viewModel: OnboardingViewModel
+
+    var body: some View {
+
+        NavigationView {
+            Form {
+                Section {
+                    Picker(
+                        "Network",
+                        selection: $viewModel.selectedNetwork
+                    ) {
+                        Text("Signet").tag(Network.signet)
+                        Text("Testnet").tag(Network.testnet)
+                    }
+                    .pickerStyle(.navigationLink)
+                    .accessibilityLabel("Select bitcoin network")
+                    .scrollContentBackground(.hidden)
+
+                    Picker(
+                        "Server",
+                        selection: $viewModel.selectedEsploraServer
+                    ) {
+                        ForEach(viewModel.availableEsploraServers, id: \.self) { esploraServer in
+                            Text(esploraServer.name).tag(esploraServer)
+                        }
+                    }
+                    .pickerStyle(.navigationLink)
+                    .accessibilityLabel("Select esplora server")
+                    .scrollContentBackground(.hidden)
+                } footer: {
+                    Text(
+                        "Set your desired network and connection server.\nIf in doubt, use the default settings."
+                    )
+                }
+            }
+            .navigationTitle("Network settings")
+            .navigationBarTitleDisplayMode(.inline)
+            .scrollContentBackground(.hidden)
+            .toolbar {
```

Comment:
change requested: remove this entire `.toolbar` because the `Back` button does not make sense to me, since when we dismiss this sheet it dismisses down (not left). 

I think we chatted about this a bit, and thought maybe a "Done" in the toolbar trailing might be the thing, so feel free to do that if you'd like but the "Back" button is something I'd definitely not keep. I'm ok with it just having a swipe to dismiss right now and we come back to the "Done" part later if we're not 100% sure what we want to do with that part at the moment.

Reply:
Instead of removing and having no visual dismiss button here I have replaced the Back button on the left with a Done button on the right, following iOS conventions.

Reply:
perfect, resolving conversation